.eslintrc.json

```diff
@@ -0,0 +1,105 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:import/errors",
+        "plugin:import/warnings"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true,
+            "jsx": true
+        }
+    },
+    "plugins":  [
+        "react"
+    ],
+    "parser": "babel-eslint",
+    "rules":  {
+    "indent": [
+      "error",
+      2, {
+        "FunctionDeclaration": {"body": 1, "parameters": 2},
+        "FunctionExpression": {"body": 1, "parameters": 2},
+        "CallExpression": {"arguments": 1},
+        "MemberExpression": 1,
+        "SwitchCase": 1
+      }
+    ],
+    "linebreak-style": [
+      "error",
+      "unix"
+    ],
+    "operator-linebreak": [
+      "error",
+      "after"
+    ],
+    "quotes": [
+      "error",
+      "single",
+      {"avoidEscape": true}
+    ],
+    "semi": [
+      "error",
+      "always"
+    ],
+    "no-trailing-spaces": [
+      "error"
+    ],
+    "space-before-function-paren": [
+      "error", {
+        "anonymous": "never",
+        "named": "never",
+        "asyncArrow": "always"
+      }
+    ],
+    "no-var": [
+      "error"
+    ],
+    "prefer-const": [
+      "error"
+    ],
+    "no-multiple-empty-lines": [
+      "error"
+    ],
+    "eol-last": [
+        "error",
+        "always"
+    ],
+    "max-len": [
+      "error",
+      82
+    ],
+    "comma-spacing": [
+      "error",
+      {"before": false, "after": true}
+    ],
+    "comma-dangle": [
+      "error",
+      "never"
+    ],
+    "no-console": [
+      "error",
+      {"allow": ["warn", "error"]}
+    ],
+    "curly": [
+      "error"
+    ],
+    "no-unused-vars": [
+      "error",
+      {"argsIgnorePattern": "^__"}
+    ],
+    "keyword-spacing": [
+      "error",
+      {"before": true,"after": true}
+    ],
+    "space-before-blocks": [
+      "error"
+    ]
+  }
```

Comment:
Please review these rules and include if possible
 - camelcase rule
- strict
- no-tabs

Reply:
@surajdevesan 'no-tabs', it is implied in the indent rule :stuck_out_tongue_winking_eye:

Reply:
@thatsmeamal no-tabs rule ensures that no tab characters are there in the code including comments and other operations 
ex:  ` var a /t= 2;` *which is only caught by no-tab rule and not indent rule.*
**Please refer the [docs](https://eslint.org/docs/rules/no-tabs#rule-details) for further clarification** :stuck_out_tongue_closed_eyes:

Reply:
updated the rule set